vue/src/components/Results.vue

```diff
@@ -126,19 +126,15 @@ export default {
     }
   },
   mounted() {
-    if (this.single) {
-      this.fetchSingleQuery(this.single)
-    } else if (this.input) {
-      this.fetchQuery(this.input)
-    }
+    console.info("Results component mounted");
+    this.forceRerender();
```

Comment:
I'll merge anyway to keep us moving - but it feels like this and the unmounted() would ideally be superfluous.  There must be a way of getting Vue to create a fresh clean Results object, e.g. React key-style.